src/core/Microsoft.PowerToys.Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -97,6 +97,13 @@ public GeneralViewModel()
             _autoDownloadUpdates = GeneralSettingsConfigs.AutoDownloadUpdates;
             _isElevated = ShellPage.IsElevated;
             _runElevated = GeneralSettingsConfigs.RunElevated;
+
+            try
+            {
+                // This fails for the Unit Tests
+                loader = ResourceLoader.GetForCurrentView();
+            }
+            catch { }
```

Comment:
Code that is specific for testing should be isolated and not effect in any way the production code.
In this case the change is not ideal. Even if in production this call doesn't fail, the code should not just catch the exception and not handle it properly.

Reply:
You are right. This was intended as a hotfix, but will delay this issue to deliver a proper solution in the future since we don't need the tests to run for the build farm at the moment. I'll close this PR.